utility/src/main/AndroidManifest.xml

```diff
@@ -1 +1,3 @@
-<manifest package="org.oppia.util"/>
+<manifest package="org.oppia.util">
```

Comment:
I think manifests need to at least have an empty application tag to be valid. It’s also a good idea to include the min and target sdk versions. Please fix.

Reply:
Done!